src/cli.rs

```diff
@@ -169,7 +169,12 @@ pub(crate) fn build_cli() -> Command {
                 .long("continue-on-errors")
                 .env("KUBEWARDEN_CONTINUE_ON_ERRORS")
                 .action(ArgAction::SetTrue)
-                .hide(true)
+                .hide(true),
+            Arg::new("otlp-endpoint")
+                .long("otlp-endpoint")
+                .env("OTEL_EXPORTER_OTLP_ENDPOINT")
```

Comment:
we could change this to `KUBEWARDEN_OTLP_ENDPOINT`if we decide to keep this cli flag